git_machete/git_operations.py

```diff
@@ -40,14 +40,14 @@ def __init__(self, cli_opts: CommandLineOptions) -> None:
         self.__contains_equivalent_tree_cached: Dict[Tuple[str, str], bool] = {}
 
     @staticmethod
-    def run_git(git_cmd: str, *args: str, **kwargs: Dict[str, str]) -> int:
+    def _run_git(git_cmd: str, *args: str, **kwargs: Dict[str, str]) -> int:
```

Comment:
Hmmm so should it be single or double underscore in private method names? 🤔 is there a difference?

Reply:
https://docs.python.org/3/tutorial/classes.html#private-variables
_var/method I create when I want to show that it's method that is meant for using for class members only, but it shouldn't break the class. __var/method I would use if it can alter the state of the class and therefore break it.